setup.py

```diff
@@ -50,10 +50,14 @@
         "jupyter",
     ]
 
+    modin_requirements = ["modin"]
+
+    modin_test_engines_requirements = ["ray", "dask"]
+
     test_requirements = [
         "pytest",
         "pytest-mock",
-    ]
+    ] + modin_test_engines_requirements
```

Comment:
I'm a bit confused. Is there a situation in which we would want to be able to install our test requirements but not the modin test requirements? If so, we aren't able to as you have it, but if we wouldn't, why not just include ray and dask in the test-requirements directly?

Reply:
I could include ray and dask in the `test_requirements` directly; the idea here was just to make it easier to see the different types of requirements. But if you think it adds confusion, I'll change it.

Reply:
I think it should be bundled unless we want to enable a person to install only test without dask/ray and be able to run the tests that don't use them. That seems like an unlikely use case?